### PR TITLE
Update M420.md to clarify units and add example

### PR DESCRIPTION
**Pull request by tareko**
Clarify units (mm) for Z value and also give working example of how to enable fade height

### CODE REVIEW

**Comment by thinkyhead** [comment]
When the machine is in inches mode, the units are given in inches.

**Comment by tareko** [comment]
When in inches, is the unit literally inches? (e.g., 1 = 1 inch)

If so, then I'll modify accordingly.

tarek : )

**Comment by tareko** [comment]
I have updated the documentation to include language on inch mode. I believe this should make the documentation more clear. I've made it a bit redundant by adding the same language three times, but considering how many people look in one place but not the other, I think this is reasonable.

**Comment by thinkyhead** [comment]
> When in inches, is the unit literally inches? (e.g., 1 = 1 inch)

Correct.

File: _gcode/M420.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ codes:
   - M420
 
 long:
-  - Get and/or set the enabled state of bed leveling compensation, plus Z fade height.
+  - Get and/or set the enabled state of bed leveling compensation, plus Z fade height. Units will be in mm unless *inch* mode is set with [`G20`](/docs/gcode/G020.html) (which requires `INCH_MODE_SUPPORT`).
```
Comment (thinkyhead) [inline _gcode/M420.md]: You think we should add this kind of note to every linear unit parameter?
A better idea is not to mention mm or inches. Just say "in current units."

**Comment by boelle** [comment]
should this then not be rejected? mold is starting to grow on it :-D

**Pull request closed without merging**